### PR TITLE
Clarify why the older policy wins in GEP 713

### LINKED ISSUE #3434: Question: why the older policy wins in GEP 713?
In https://github.com/kubernetes-sigs/gateway-api/blob/main/geps/gep-713/index.md#conflict-resolution:

> The oldest Policy based on creation timestamp beats a newer one. For example, a Policy with a creation timestamp of "2021-07-15 01:02:03" MUST be given precedence over a Policy with a creation timestamp of "2021-07-15 01:02:04".

Naturally, newer is better. I'd like to know why the authors of GEP 713 designed a mechanism that the older policy will beat a newer one.

### PR DESCRIPTION
**Pull request by spacewander**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Clarify why the older policy wins in GEP 713

**Which issue(s) this PR fixes**:

Fixes #3434

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: geps/gep-713/index.md [hunk 1]
```diff
@@ -246,9 +246,12 @@ ties:
...
   `defaults`.
-* The oldest Policy based on creation timestamp beats a newer one. For example,
-  a Policy with a creation timestamp of "2021-07-15 01:02:03" MUST be given
-  precedence over a Policy with a creation timestamp of "2021-07-15 01:02:04".
+* The older Policy based on creation timestamp beats a newer one. For example,
+  a Policy with a creation timestamp of "2021-07-15 01:02:03" MUST be given
+  precedence over a Policy with a creation timestamp of "2021-07-15 01:02:04".
+  The goal is to ensure that introducing new, unused policies doesn’t disrupt
+  existing ones, since changing active rules can cause outages while altering
+  unused policies poses no risk.
```
Comment (FelipeYepez) [inline geps/gep-713/index.md:254]:
```suggestion
  Modifying an active Policy carries the risk of causing outages or unexpected behavior while a new Policy that hasn't been applied yet won't cause immediate risk. 
```

**Review by FelipeYepez — COMMENTED** [review]
Consider this suggestions which can be improved if needed

File: geps/gep-713/index.md [hunk 2]
```diff
@@ -246,9 +246,12 @@ ties:
   only come up in exceptional circumstances.
 * Inside Inherited Policies, the same setting in `overrides` beats the one in
   `defaults`.
-* The oldest Policy based on creation timestamp beats a newer one. For example,
-  a Policy with a creation timestamp of "2021-07-15 01:02:03" MUST be given
-  precedence over a Policy with a creation timestamp of "2021-07-15 01:02:04".
+* The older Policy based on creation timestamp beats a newer one. For example,
```
[the remark this answers is not in the document]
Comment (youngnick) [reply geps/gep-713/index.md:249]: I think either option is fine. @spacewander's current version reads more like the existing comments, but both version are clear.

**Comment by youngnick** [comment]
Either option here LGTM, so I'll approve.

/approve

**Review by kflynn — APPROVED** [review]
I'd vote to accept @spacewander's current text. All due respect to @FelipeYepez (🙂), I prefer the version that clearly calls out the goal.

Thanks @spacewander! 🙂

**Comment by FelipeYepez** [comment]
Sure, removed suggestion. Thanks and sorry

**Comment by kflynn** [comment]
@FelipeYepez No need to apologize!! Your suggestions were good and it was helpful to think about what I, personally, might find most helpful in the docs, others might disagree with my opinion there. 🙂

**Comment by youngnick** [comment]
@FelipeYepez, @kflynn has it absolutely right - thanks for your suggestions! Please don't feel like your effort was wasted, the more folks we have look at this stuff, the better.

**Pull request merged**